### PR TITLE
Digi_32_UniqueStoreTitle 

### PR DESCRIPTION
**Pull request by rajendarreddy1234**
1.Added StoreController
2.Added StoreDataModel

### CODE REVIEW

File: DigiOrderApi/DigiOrderApi/Controllers/StoreController.cs [hunk 1]
```diff
@@ -14,30 +15,39 @@ public class StoreController : ApiController
...
 
         /// <summary>
-        /// Author: Rajendar Reddy konuganti
+        /// Author: Rajendar Reddy Konuganti
         /// Date: 7/14/2020
-        /// Description: Get all available Stores from database
-        /// Ref. StoryId: DIGI 2
-        /// Reviewer: Bhargav Krishna A
+        /// Description:Inserting Stores into Database
+        /// Ref. StoryId: DIGI 1
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApi/Controllers/StoreController.cs:21]: add unique story id as well

Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApi/Controllers/StoreController.cs:21]: Add unique story id as well

Comment (rajendarreddy1234) [reply DigiOrderApi/DigiOrderApi/Controllers/StoreController.cs]: Resolved

Comment (rajendarreddy1234) [reply DigiOrderApi/DigiOrderApi/Controllers/StoreController.cs]: Resolved

**Pull request merged**